App.cs

```diff
@@ -0,0 +1,101 @@
+
+using System.Net.Http.Headers;
+using System.Text;
+
+class App {
+  BattleGround player1;
+  BattleGround player2;
```

Comment:
Looks weird when player is a battleground. Create new class for players and wrap that

Reply:
fixed